src/Core/Query/LocalParameters/LocalParameter.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Solarium\Core\Query\LocalParameters;
+
+/**
+ * Local Parameter.
+ *
+ * @author wicliff <wicliff.wolda@gmail.com>
+ */
+class LocalParameter implements LocalParameterInterface
+{
+    public const TYPE_KEY = 'key';
+
+    public const TYPE_EXCLUDE = 'ex';
+
+    public const TYPE_RANGE = 'range';
+
+    public const TYPE_TAG = 'tag';
+
+    public const TYPE_TERM = 'terms';
+
+    public const TYPE_QUERY = 'query';
+
+    public const TYPE_STAT = 'stats';
+
+    public const TYPE_MIN = 'min';
+
+    public const TYPE_MAX = 'max';
+
+    public const TYPE_MEAN = 'mean';
+
+    public const TYPE_DEFAULT_FIELD = 'df';
+
+    public const TYPE_QUERY_FIELD = 'qf';
+
+    public const TYPE = 'type';
```

Comment:
Wouldn't TYPE_TYPE be more consistent?

Reply:
yes you're absoletely right; will modify.